apps/web/src/components/ActionButtons.tsx

```diff
@@ -40,6 +41,7 @@ export const ActionButtons = () => {
           colorScheme="purple"
           leftIcon={<SiSubstack />}
           mt={{ base: '0', md: '10px' }}
+          width="100%"
```

Comment:
Flex 1?

Reply:
Makes sense, will fix it!

Reply:
Maybe not
![image](https://user-images.githubusercontent.com/29582786/151077938-a0e5e757-d4a5-4ecd-911a-8eafdcb548b8.png)

Reply:
I think that you can do something like it:

```jsx
// ...
<VStack>
  // Tweet button
  <HStack>
    // Social Buttons here
  </HStack>
  // Other buttons
</VStack>

Reply:
`width: 100%` seems better to me